notebooks/templates/ai_platform_notebooks_template.ipynb

```diff
@@ -1,524 +1,634 @@
 {
```

Comment:
<p>Is there a missing tick for markdown around the ###?</p><p> </p><p>Also, is something missing around "1. into these commands?"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>That's an existing cell. I made no changes. Not sure what you mean by extra tick around ###. I do see an extra period in 1. -- so I remove it.</p>